suite-native/accounts/src/components/AccountsListGroup.tsx

```diff
@@ -1,23 +1,28 @@
-import { Card, VStack } from '@suite-native/atoms';
-import { Account, AccountKey, TokenAddress } from '@suite-common/wallet-types';
+import { Account } from '@suite-common/wallet-types';
+import { VStack } from '@suite-native/atoms';
 
+import { OnSelectAccount } from '../types';
 import { AccountListItemInteractive } from './AccountListItemInteractive';
 
 type AccountsListGroupProps = {
     accounts: Account[];
-    onSelectAccount: (accountKey: AccountKey, tokenContract?: TokenAddress) => void;
+    onSelectAccount: OnSelectAccount;
+    hideTokens?: boolean;
 };
 
-export const AccountsListGroup = ({ accounts, onSelectAccount }: AccountsListGroupProps) => (
-    <Card>
-        <VStack spacing="medium">
-            {accounts.map(account => (
-                <AccountListItemInteractive
-                    key={account.key}
-                    account={account}
-                    onSelectAccount={onSelectAccount}
-                />
-            ))}
-        </VStack>
-    </Card>
+export const AccountsListGroup = ({
```

Comment:
something rebuilds dashboard quite often in this PR


https://github.com/user-attachments/assets/2fc73b2e-08f0-4eea-9434-64b15da2f03f

Reply:
Can you please share xpubs for that ETC accounts? It may be related.